routes/auth.js

```diff
@@ -21,12 +21,15 @@ const isUser = async function(req, res, next) {
       res.send({message: "User not authorized, invalid token"})
     }
   }
+  else {
+    res.send({message: "User not authorized, invalid token"})
+  }
 }
 
 const isAdmin = async function(req, res, next) {
   const token = req.header('Authorization')
   if(token) {
-    const user = await userModel.findOne({token: token},{admin: true})
+    const user = await userModel.findOne({ $and:[{token: token},{admin: true}]})
```

Comment:
Why not just use `await userModel.findOne({ token, admin: true })`?

Am I missing something? :thinking:

Reply:
Will make the change